common/app/model/Section.scala

```diff
@@ -63,3 +63,31 @@ object SectionId {
 
   def fromId(id: String): SectionId = SectionId(id)
 }
+
+object Neilsen {
+  private val sectionIdToApid = Map(
```

Comment:
should this be `sectionIdToAppId`? or is apid a thing in itself?

Reply:
APIDs are unique ids used by Nielsen. I'm not sure what it stands for :P